docs/reference/spec.md

```diff
@@ -49,7 +49,7 @@ e.g `authorization/foo-org-xyz` references the Foo Organization.
 
 ## Inception
```

Comment:
In general, we should plan to remove this spec.md, it was originally a brain dump for how this should work, and I think the new documentation should supersede it. So I suggest if there's anything of value still in here that it be worked in to the main documentation somehow.